app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -98,14 +98,16 @@ public VideoManager(PlaybackOverlayActivity activity, View view) {
         mSubtitlesSurface.setZOrderMediaOverlay(true);
         mSubtitlesSurface.getHolder().setFormat(PixelFormat.TRANSLUCENT);
 
-        mExoPlayer = new ExoPlayer.Builder(TvApp.getApplication(), new DefaultRenderersFactory(TvApp.getApplication()) {
-            @Override
-            protected void buildTextRenderers(Context context, TextOutput output, Looper outputLooper, int extensionRendererMode, ArrayList<Renderer> out) {
-                // Do not add text renderers since we handle subtitles
-            }
-        }).build();
-
-
+        DefaultRenderersFactory defaultRendererFactory =
+                new DefaultRenderersFactory(TvApp.getApplication()) {
+                    @Override
+                    protected void buildTextRenderers(Context context, TextOutput output,
+                                                      Looper outputLooper, int extensionRendererMode,
+                                                      ArrayList<Renderer> out) {
+                        // Do not add text renderers since we handle subtitles
+                    }
+                }.setExtensionRendererMode(DefaultRenderersFactory.EXTENSION_RENDERER_MODE_PREFER);
+        mExoPlayer = new ExoPlayer.Builder(activity.getApplicationContext(), defaultRendererFactory).build();
```

Comment:
I think getApplicationContext() still gives the same app-level context you replaced.

The requested change was to use the activity context

Reply:
It was mostly to not use the static getApplication function (which we shouldn't use in new code), so this should be fine.

Reply:
I didn't realise that `activity` was implicitly a context, I was looking for a `getContext` or `requireContext` method.
This should be resolved now :+1: